packages/web/src/pages/sign-up-page/utils/passwordSchema.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from 'zod'
+
+import { isNotCommonPassword } from 'utils/commonPasswordCheck'
+
+export const passwordSchema = z
+  .object({
+    password: z
+      .string()
+      .regex(/\d/, { message: 'hasNumber' })
```

Comment:
are these keys for the messages object in CreatePasswordPage? Should we type them to match?

Reply:
yeah they are! i agree this should be a type